libs/blocks/bulk-publish-v2/services.js

```diff
@@ -80,15 +77,18 @@ const isPushedDown = async () => {
 
 const authenticate = async (tool = null) => {
   isPushedDown();
-  const setUser = (event) => { tool.user = setUserData(event); };
+  const setUserV6 = (event) => { tool.user = setUserData(event?.detail?.data); };
+  const setUserV7 = (event) => { tool.user = setUserData(event?.detail); };
   const openSideKick = document.querySelector('aem-sidekick, helix-sidekick');
   if (openSideKick) {
-    openSideKick.addEventListener('statusfetched', setUser);
+    openSideKick.addEventListener('statusfetched', setUserV6); // sidekick v6
+    openSideKick.addEventListener('status-fetched', setUserV7); // sidekick v7
```

Comment:
@mokimo - would we also need to take care of:
* https://github.com/adobecom/milo/blob/03e75059ae3643e0699eac49598925ede956b91c/libs/utils/sidekick-decorate.js#L86
* https://github.com/adobecom/milo/blob/03e75059ae3643e0699eac49598925ede956b91c/libs/blocks/preflight/panels/general.js#L85
I guess this can be done in a subsequent PR

Reply:
The decorate logic should already account for the new event https://github.com/adobecom/milo/pull/3107/files#diff-8fca44a004d5ac9b5fd40efac647ac0deb02ada1314e83f34593176290dcbb53R169-R176

Reply:
Added a check for the preflight